lib/nerves_hub/extensions/extensions_setting.ex

```diff
@@ -1,15 +1,61 @@
 defmodule NervesHub.Extensions.ExtensionsSetting do
   use Ecto.Schema
   import Ecto.Changeset
+  @behaviour Access
```

Comment:
```suggestion
  @derive [Access]
```
You might be able to just derive Access and avoid all the extra implementation if that is what you're aiming for

Reply:
I looked for whatever the heck the intended way is and could not find it :D

Reply:
Ach, access is not a protocol, it is a behaviour.

Reply:
You know, that makes sense. I was going off this article which is 10 years old....clearly it has changed since then 🤣 https://dashbit.co/blog/writing-assertive-code-with-elixir